lib/configuration/variables/sources/instance-dependent/get-aws.js

```diff
@@ -21,10 +21,9 @@ module.exports = (serverlessInstance) => {
 
       switch (address) {
         case 'accountId': {
-          const { Account } = await serverlessInstance
-            .getProvider('aws')
-            .request('STS', 'getCallerIdentity', {}, { useCache: true });
-          return { value: Account };
+          const provider = serverlessInstance.getProvider('aws');
+          const result = await provider.request('STS', 'getCallerIdentity', {}, { useCache: true });
+          return { value: result.Account };
```

Comment:
Can we revert formatting changes like these? This will make the diff much smaller, and also will minimize changes with upstream. It seems like there are 4-5 files impacted.